charts/retool/templates/_workers.tpl

```diff
@@ -0,0 +1,413 @@
+{{- define "retool.workers" -}}
+- parent: agents
+  type: agent
+- parent: agents
+  type: agentEval
+- parent: workflows
+  type: workflow
+{{- end -}}
+
+{{- define "retool.worker.deployments" -}}
+{{- $root := . -}}
+{{- $workers := include "retool.workers" . | fromYamlArray -}}
+
+{{- range $worker := $workers -}}
+{{- if eq (include (printf "retool.%s.enabled" $worker.parent) $root) "1" -}}
+{{ include "retool.worker.deployment" (dict "root" $root "parent" $worker.parent "workerType" $worker.type) }}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "retool.worker.deployment" -}}
+{{- $ := .root -}}
+{{- $parent := .parent -}}
+{{- $workerType := .workerType -}}
+{{- $parentValues := index $.Values $parent -}}
+
+{{- $workerValues := $parentValues.worker -}}
+{{- if eq $workerType "agentEval" -}}
+  {{- $workerValues = $parentValues.evalWorker -}}
+{{- end -}}
+
+{{- $healthcheckPort := ternary 3012 3005 (eq $workerType "agentEval") -}}
```

Comment:
why does the agentEvalWorker have a different healthcheck port than the rest? I believe we also normally allow folks to adjust these, as sometimes people don't have the right ports free. Is it possible to have defaults in values.yaml instead?